src/encoded/audit/experiment.py

```diff
@@ -290,18 +296,17 @@ def audit_experiment_out_of_date_analysis(value, system, files_structure):
                   'transcriptome_alignments']
     for file_type in file_types:
         for bam_file in files_structure.get(file_type).values():
-            if bam_file.get('lab') == '/labs/encode-processing-pipeline/' and \
-                    bam_file.get('derived_from'):
-                if is_outdated_bams_replicate(bam_file, files_structure):
+            if bam_file.get('lab') == '/labs/encode-processing-pipeline/' and bam_file.get('derived_from'):  
+                if is_outdated_bams_replicate(bam_file, files_structure, assay_name):
                     assembly_detail = ''
                     if bam_file.get('assembly'):
                         assembly_detail = ' for {} assembly '.format(
                             bam_file['assembly'])
-                        detail = 'Experiment {} '.format(value['@id']) + \
+                    detail = 'Experiment {} '.format(value['@id']) + \
```

Comment:
Out of scope but this should be fixed at some point.
```
detail = 'Experiment {} '.format(value['@id']) + \
'alignment file {} '.format(
bam_file['@id']) + assembly_detail + \
'is out of date.'
```
Something like:
```
# Now one space by default.
assembly_detail = ' '

. . .

detail = 'Experiment {} alignment file {}{}is out of date.'.format(value['@id'],
                                                                   bam_file['@id'],
                                                                   assembly_detail)

```

Reply:
fixed